pages/common/xml-unescape.md

```diff
@@ -0,0 +1,16 @@
+# xml-unescape
+
+> Un-escape special XML characters.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Unescape special XML characters from a string:
+
+`xml unescape {{"&lt;a1&gt;"}}`
+
+- Unescape special XML characters from standard input:
```

Comment:
Perhaps this will read better?
```suggestion
- Unescape special XML characters from the estandard input:
```

Reply:
The majority of existing examples do not use "the" before "standard input". These "xml" commands will now be most of those that do.

❯ grep "standard input" *.md
at.md:- Execute commands from standard input in 5 minutes (press `Ctrl + D` when done):
at.md:- Execute a command from standard input at 10:00 AM today:
base32.md:> Encode or decode file or standard input to/from Base32, to standard output.
base64.md:> Encode or decode file or standard input to/from Base64, to standard output.
batch.md:- Execute commands from standard input (press `Ctrl + D` when done):
batch.md:- Execute a command from standard input:
carbon-now.md:- Create an image from standard input using default settings:
cmark.md:- Convert data from standard input to LaTeX:
column.md:> Format standard input or a file into multiple columns.
cpio.md:- Take a list of file names from standard input and add them [o]nto an archive in cpio's binary format:
expand.md:- Convert tabs to spaces, reading from standard input:
gh-auth.md:- Log in with a token from standard input (created in https://github.com/settings/tokens):
git-column.md:- Format the standard input as multiple columns:
git-column.md:- Format the standard input as multiple columns with a maximum width of `100`:
git-column.md:- Format the standard input as multiple columns with a maximum padding of `30`:
git-stripspace.md:> Read text (e.g. commit messages, notes, tags, and branch descriptions) from the standard input and clean it into the manner used by Git.
mail.md:> To send an email the message body is built from standard input.
md5sum.md:- Calculate a MD5 checksum from the standard input:
msmtp.md:> It reads text from standard input and sends it to an SMTP server.
nl.md:> A utility for numbering lines, either from a file, or from standard input.
read.md:> BASH builtin for retrieving data from standard input.
stdbuf.md:- Change the standard input buffer size to 512 KiB:
tac.md:- Print the contents of the standard input reversed to the standard output:
tee.md:> Read from standard input and write to standard output and files (or commands).
tee.md:- Copy standard input to each FILE, and also to standard output:
tee.md:- Print standard input to the terminal, and also pipe it into another program for further processing:
wc.md:- Use standard input to count lines, words and characters (bytes) in that order:

Reply:
xml.md:- Execute a subcommand with input from a file or URI, printing to stdout:
xml.md:- Execute a subcommand using stdin and stdout:
xml-depyx.md:- Convert a PYX document from stdin to XML format:
xml-escape.md:- Escape special XML characters from stdin:
xml-format.md:- Format an XML document from stdin, removing the DOCTYPE declaration:
xml-pyx.md:- Convert an XML document from stdin to PYX format:
xml-unescape.md:- Unescape special XML characters from stdin: